app/main.js

```diff
@@ -1,163 +1,229 @@
-const { app, BrowserWindow, ipcMain, shell } = require('electron');
+const electron = require('electron');
+const {
+    app,
+    BrowserWindow,
+    ipcMain, 
+    Menu, 
+    shell
+} = electron;
 const fs = require('fs-extra');
 const storage = require('electron-json-storage');
 const _ = require('lodash');
 const SWProxy = require('./proxy/SWProxy');
-
 const path = require('path');
 const url = require('url');
-
 global.gMapping = require('./mapping');
-
 global.win;
 let defaultFilePath = path.join(app.getPath('desktop'), app.getName() + ' Files');
+let template = [{
+  label: 'Window',
+  role: 'window',
+  submenu: [{
+    label: 'Minimize',
+    accelerator: 'CmdOrCtrl+M',
+    role: 'minimize'
+  }]
+}, {
+  label: 'Help',
+  role: 'help',
+  submenu: [{
+    label: "Summoner's War Exporter GitHub",
+    click: function () {
+      shell.openExternal('https://github.com/Xzandro/sw-exporter')
+    }
+  }]
+}]
+
+function findReopenMenuItem () {
```

Comment:
I got it from "https://github.com/electron/electron-api-demos". Supposed to be to re-open the window after closing it (with "command-W" on Mac for example). I got the window closing no problem but it wouldn't seem to open back up so I just got rid of the close/re-open as well as this function